locale/core.yml

```diff
@@ -42,6 +42,13 @@ core:
       support_text: "Found a bug? Please report it in our forum, under the <a>Support tag</a>."
       troubleshooting_text: "Having problems? Follow the instructions in the <a>Troubleshooting docs</a>."
       version_text: "Thanks for trying out Flarum! You are running version {version}."
+      welcome_text: "Welcome to Flarum Beta!"
+
+    # These translations are used in the Edit Custom CSS modal dialog.
+    edit_css:
+      customize_text: Customize your forum's appearance by adding your own LESS/CSS code to be applied on top of Flarum's <a>default styles</a>.
+      submit_button: => core.ref.save_changes
+      title:
```

Comment:
Yeah, sorry, let's be clearer: this key is alone, without translation after it. Do you missed a reused key or the translation? Because for now, this key is useless.